xscen/indicators.py

```diff
@@ -291,3 +290,55 @@ def func(ds, *, ind, nout):
 
     func.__name__ = ind.identifier
     return partial(func, ind=ind, nout=nout)
+
+
+def select_inds_for_avail_vars(
+    ds: xr.Dataset,
+    indicators: Union[
+        str, Path, Sequence[Indicator], Sequence[tuple[str, Indicator]], ModuleType
```

Comment:
`os.PathLike` is more robust than just `Path`.
```suggestion
        str, os.PathLike, Sequence[Indicator], Sequence[tuple[str, Indicator]], ModuleType
```

Reply:
Fixed.